backend/app/data_layer/database/db_connections/postgresql.py

```
@@ -0,0 +1,24 @@
+from sqlmodel import create_engine, SQLModel, Session, select
```

Ruff (0.8.0)

`sqlmodel.select` imported but unused

Remove unused import: `sqlmodel.select`

(F401)